logins.js

```diff
@@ -25,10 +25,16 @@ module.exports = {
                 return callback(err, null);
             }
             
+            if (body.trim().indexOf('<') === 0) {
```

Comment:
Yup, didn't notice that. You are right. I don't think it will be an issue to just check the first position in the string.